src/Migration.php

```diff
@@ -1716,6 +1716,23 @@ private function migrateSearchOptions()
                     'itemtype' => $itemtype,
                     'num'      => $old_search_opt
                 ]);
+
+                // Update template fields
+                $item = new $itemtype();
+                if ($item instanceof CommonITILObject) {
+                    $tables = [
+                        'glpi_' . $itemtype . 'templatehiddenfields',
+                        'glpi_' . $itemtype . 'templatemandatoryfields',
+                        'glpi_' . $itemtype . 'templatepredefinedfields',
+                    ];
+                    foreach ($tables as $table) {
```

Comment:
This method can be used by a plugin migration on a class that extends the CommonITILObject class without implementing the template fields feature (for instance the `PluginReleasesRelease` class).

```suggestion
                    foreach ($tables as $table) {
                        if (!$DB->tableExists($table)) {
                            continue;
                        }
```